### PR TITLE
Fix hystrix.latency.execution and hystrix.latency.total

### PR DESCRIPTION
**Pull request by izeye**
`hystrix.latency.execution` and `hystrix.latency.total` are swapped, so this PR fixes it.

This PR also introduces constants to avoid duplicate `String` literals and local variables having `String` literals. But some of them are just for consistency.

### CODE REVIEW

File: micrometer-core/src/main/java/io/micrometer/core/instrument/binder/hystrix/MicrometerMetricsPublisherCommand.java [hunk 1]
```diff
@@ -103,7 +118,7 @@ public void initialize() {
                 }
                 long executionLatency = hystrixCommandCompletion.getExecutionLatency();
                 if (executionLatency >= 0) {
-                    Timer.builder("hystrix.latency.total")
+                    Timer.builder(NAME_HYSTRIX_LATENCY_EXECUTION)
```
Comment (checketts) [inline micrometer-core/src/main/java/io/micrometer/core/instrument/binder/hystrix/MicrometerMetricsPublisherCommand.java]: How the heck are you finding all these? Much appreciated!

Comment (jkschneider) [reply micrometer-core/src/main/java/io/micrometer/core/instrument/binder/hystrix/MicrometerMetricsPublisherCommand.java]: I want to know the same thing.

Comment (izeye) [reply micrometer-core/src/main/java/io/micrometer/core/instrument/binder/hystrix/MicrometerMetricsPublisherCommand.java]: @checketts @jkschneider Just by watching this project 😅

Comment (spencergibb) [reply micrometer-core/src/main/java/io/micrometer/core/instrument/binder/hystrix/MicrometerMetricsPublisherCommand.java]: If you haven't been following the work done by @izeye in spring-boot, do, you'll be even more amazed.

**Pull request merged**